### DISCUSSION TITLE
Issue template and MS Store price....

### DISCUSSION THREAD

**Discussion by PySimpleGUI** [infinitepower18/WSA-Sideloader#13] [General]
Hiya!  Two questions for you. 

One is how you did you set up the Issues the way you've done them with the templates / forms?  I've not seen any other Repos use Issues like this.  I don't recall GitHub Issues as having advanced features like what I see here or else I wouldn't have made the templates for the PySimpleGUI repo.

The other is..... what price are you going to set for WSA-Sideloader in the Microsoft Store?  Looks like a sellable idea  ![image](https://user-images.githubusercontent.com/46163555/157337504-c6a9645d-911b-4598-be3f-7adee5246dbf.png)

**Comment by infinitepower18** [comment]
Hi, WSA Sideloader is a free program, I have no plans to make it a paid program or put any ads on it. Besides, similar programs like WSA Pacman and WSATools are also free.

I actually noticed [Microsoft PowerToys](https://github.com/microsoft/powertoys) use this new issue format, and after searching around on how to do it, I came across this in the GitHub documentation: https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#creating-issue-forms. Sadly, this kind of format isn't available for pull request template as of now.

I hope this helps and thank you for checking out my program and making PySimpleGUI, it really made things easier for me being a computer science student currently and trying to improve my Python skills (I learnt a lot when making this 😊)

**Comment by PySimpleGUI** [comment]
> I learnt a lot when making this 

You're on an excellent path.  I think the best way to learn to program is by programming.  It's a craft that requires practice.  At least that's been how I've experienced the process of learning to program.  I started learning Python in Oct 2017 and learned the most by building something.  I learned Python because I needed to build something.   

I really like what you've done here on GitHub.  It's clear you've got good organizational skills.   You're a CS student now but have had a GitHub account since 2018?

**Reply by infinitepower18** [reply]
Yeah, had GitHub since 2018 but back then I mostly used it to report bugs and make some small contributions. WSA Sideloader is the first project I made completely by myself, right now Windows Subsystem for Android is in a limited preview phase and if you're outside of the US you have to go through some workarounds to install the subsystem (as mentioned in the FAQ) but once Microsoft opens it up to everyone I'm sure tools like WSA Sideloader will gain more popularity, as without such program you would have type a bunch of commands each time you want to install an app outside the Amazon Appstore (which is the only supported storefront for WSA, no Play Store)

**Comment by PySimpleGUI** [comment]
>  WSA Sideloader is the first project I made completely by myself

No way!

![image](https://user-images.githubusercontent.com/46163555/157344552-97575395-2c86-4bad-bd47-f691eee7d2d6.png)

Really?  A PySimpleGUI-based project is your first one.  That's so cool!

![image](https://user-images.githubusercontent.com/46163555/157344387-59e9b0f1-2930-42ce-a2ea-76b5030e8323.png)

Thrilled to get to be a part of that experience.  PySimpleGUI was my first Python project so we've got something in common with firsts.  You're off and running with a strong start!

**Reply by infinitepower18** [reply]
First open source GitHub project to be precise, I worked on some other Python projects as part of my course. WSA Sideloader is the first time I ever used PySimpleGUI though. Might release my other work on GitHub in the future. Thanks for your support I really appreciate it! 😊

**Reply by PySimpleGUI** [reply]
> Thanks for your support I really appreciate it! 

It's all circular!  I get just as much from the support you give to PySimpleGUI... round and round it goes.

Can you drop me an email?  I don't see any listed in your top-level account.

**Reply by infinitepower18** [reply]
Sent you a mail
